docs/buildtest_tutorial_examples/spack/inspect/install_specs.txt

```diff
@@ -1,53 +1,43 @@
-git clone https://github.com/spack/spack                                                                                                                                                                                                   
-source ./spack/share/spack/setup-env.sh                                                                                                                                                                                                    
-spack install  zlib                                                                                                                                                                                                                        
+Test Script: /home/spack/runs/generic.local.bash/install_specs/clone_spack_and_install_zlib/7d2b08e2/clone_spack_and_install_zlib.sh
+Build Script: /home/spack/runs/generic.local.bash/install_specs/clone_spack_and_install_zlib/7d2b08e2/clone_spack_and_install_zlib_build.sh
+Output File: /home/spack/runs/generic.local.bash/install_specs/clone_spack_and_install_zlib/7d2b08e2/clone_spack_and_install_zlib.out
+Error File: /home/spack/runs/generic.local.bash/install_specs/clone_spack_and_install_zlib/7d2b08e2/clone_spack_and_install_zlib.err
+Log File: /home/spack/buildtest/var/logs/buildtest_hupirc5k.log
+────────────────────────────────────────────── Output File: /home/spack/runs/generic.local.bash/install_specs/clone_spack_and_install_zlib/7d2b08e2/clone_spack_and_install_zlib.out ──────────────────────────────────────────────
+                                                                                                                                                                                                                                   
+─────────────────────────────────────────────── Test File: /home/spack/runs/generic.local.bash/install_specs/clone_spack_and_install_zlib/7d2b08e2/clone_spack_and_install_zlib.sh ────────────────────────────────────────────────
+#!/bin/bash                                                                                                                                                                                                                        
+git clone https://github.com/spack/spack                                                                                                                                                                                           
+source ./spack/share/spack/setup-env.sh                                                                                                                                                                                            
+spack install  zlib                                                                                                                                                                                                                
```

Comment:
okay this is a test failure and i think we need to open an issue and rewrite the way this test is to be done. I think part of the problem is we have a dynamic test since we are using `git clone https://github.com/spack/spack` to perform an installation of zlib and this test will change depending on state of spack when we run this test. I think a better way for this would be to use a tag branch to force the test to use the same version. @Mendi03 can you please record this as an issue and we can fix this later. For time being let's keep this bug in the docs so we know we have to fix it

Reply:
I think this is similar to the issue I talked about in [#1518](https://github.com/buildtesters/buildtest/issues/1518). Should I simply update this issue mentioning that  it might be causing the error in [docs/buildtest_tutorial_examples/spack/inspect/install_specs.txt](https://github.com/buildtesters/buildtest/pull/1538/files/c7a6ed0367e7e97b65b7504e91b01294a9bd74da#diff-b9139ff4abd807189e44476b624b51f3b4c05dd709ecc003cbd728e7481a7047)? Or do I create a new issue anyways?